onnxruntime/core/framework/ort_value_pattern_planner.cc

```diff
@@ -13,6 +13,18 @@ OrtValuePatternPlanner::OrtValuePatternPlanner(const ExecutionPlanBase& executio
   }
 }
 
+common::Status OrtValuePatternPlanner::TraceAllocation(int ort_value_idx, size_t program_counter_start, size_t program_counter_end, size_t size) {
+  // TODO(codemzs): refactor code.
+  auto location = execution_planner_.GetLocation(ort_value_idx);
+  auto it = planner_map_.find(location);
+  if (it == planner_map_.end()) {
+  return common::Status(common::ONNXRUNTIME, common::INVALID_ARGUMENT);
```

Comment:
(nit):  can you format the files?  clang format config is checked in, so you can run `alt-shift-F` in VS Code to do it for you :)